pkg/role_binding/role_binding.go

```diff
@@ -119,10 +128,33 @@ func PresentRoleBinding(rb *v1.RoleBinding) RoleBinding {
 	}
 }
 
-func PresentRoleBindings(roleBindings []*v1.RoleBinding) []string {
-	var roleList []string
+func PresentRoleBindings(roleBindings []*v1.RoleBinding) []AccountRoleBinding {
+	keySeparator := "__"
+	uniqueRoleBindingsMap := make(map[string]int)
+	var uniqueRoleList []AccountRoleBinding
 	for _, roleBinding := range roleBindings {
-		roleList = append(roleList, roleBinding.Role().ID())
+		uniqueRoleBindingsKey := roleBinding.Role().ID() + keySeparator + roleBinding.Type()
+		if val, ok := uniqueRoleBindingsMap[uniqueRoleBindingsKey]; ok {
+			uniqueRoleBindingsMap[uniqueRoleBindingsKey] = val + 1
+		} else {
+			uniqueRoleBindingsMap[uniqueRoleBindingsKey] = 1
+		}
+	}
+	for k := range uniqueRoleBindingsMap {
+		keySegments := strings.Split(k, keySeparator)
+		var totalOccs *int
+		if uniqueRoleBindingsMap[k] != 1 {
```

Comment:
Although != 1 would work, semantically what we want to test here is if this is > 1.